packages/shared/lib/sdk/sync.ts

```diff
@@ -763,7 +771,12 @@ export class NangoAction {
         const level = userDefinedLevel?.level ?? 'info';
 
         if (this.dryRun) {
-            logger[logLevelToLogger[level] ?? 'info'].apply(null, args as any);
+            // TODO: control this logger inside dryrun instead
+            if (args.length > 1 && 'type' in args[1] && args[1].type === 'http') {
+                logger[logLevelToLogger[level] ?? 'info'].apply(null, [args[0], { status: args[1]?.response?.code || 'xxx' }] as any);
+            } else {
+                logger[logLevelToLogger[level] ?? 'info'].apply(null, args as any);
+            }
```

Comment:
Would something like this be easier to read
```
const status = args[1]?.type == 'http' ? args[1].response?.code: undefined
const logArgs =  status ? [args[0], { status }] : args
logger[logLevelToLogger[level] ?? 'info'].apply(null, logArgs as any);
```

Reply:
honestly I don't know ahah
I have split the repetitive part